.travis.yml

```diff
@@ -3,9 +3,9 @@ os:
   - linux
 #  - osx
 julia:
-  - 0.6
+  - 0.7
```

Comment:
If we're keeping 0.6 below, keep it here.

Reply:
ok